packages/npm-lit/demo/index.ts

```diff
@@ -0,0 +1,69 @@
+import { atom, Atom, peek } from '@reatom/core'
+import { withReatomElement, watch } from '../src/index.js'
+import { LitElement, html } from 'lit'
+
+const timer = atom(0)
+const countAtom = atom(0)
+
+const increment = atom(() => {
+  countAtom(countAtom() + 1)
+})
+
+setInterval(() => {
+  timer(timer() + 1)
```

Comment:
Just a better codestyle
```suggestion
  timer((state) => state + 1)
```

Reply:
done